src/pay/batch_payer.py

```diff
@@ -85,42 +94,43 @@ def __init__(
         self.delegator_pays_xfer_fee = delegator_pays_xfer_fee
 
         # If delegator pays the fee, then the cutoff should be transaction-fee + 1
+        # Fixed value can only be used to determine threshold for tz addresses
         # Ex: Delegator reward is 1800 mutez, txn fee is 1792 mutez, reward - txn fee = 8 mutez payable reward
         #     If delegate pays fee, then cutoff is 1 mutez payable reward
         if self.delegator_pays_xfer_fee:
             self.zero_threshold += self.default_fee
 
         logger.info(
-            "Transfer fee is {:.6f} XTZ and is paid by {}".format(
+            "Transfer fee is {:.6f} tez for tz addresses and is paid by {}.".format(
                 self.default_fee / MUTEZ,
                 "Delegator" if self.delegator_pays_xfer_fee else "Delegate",
             )
         )
         logger.info(
-            "Reactivation fee is {:.6f} XTZ and is paid by {}".format(
+            "Reactivation fee (burn fee) for tz addresses is {:.6f} tez and is paid by {}.".format(
```

Comment:
Why change XTZ to 'tez'?

Reply:
Because it is not a ticker symbol here

Reply:
ok, I realize that is wrong all over the place :D see here: https://wiki.tezosagora.org/learn/basics/nomenclature